README.md

```diff
@@ -1,135 +1,410 @@
-    ERC-20 token with support for burn-mint style transfers between the EVM
-    chains in the network.
--   An ethers projects that provides BIP-44 wallets for all allocations in the
-    Genesis block.
--   Docker compose project for running [Blockscout](https://blockscout.com)
-    instances for the EVM chains in the development network. Blockscout is a
-    blockchain explorer that provides an user experience and API similar to
-    [Etherscan](https://etherscan.io).
--   A commandline tool for managing and monitoring the different network
-    components in the Kadena EVM sandbox.
+```sh
+./network devnet restart
+```
 
-## System requirements
+### Stopping the development network
 
-For running the development network:
+When you're finished testing, you can shut down the development network, remove all containers, and reset the database to a clean state.
 
--   [docker](https://www.docker.com) or a compatible alternative.
--   [docker compose](https://docs.docker.com/compose/) or a compatible
-    alternative.
--   At least 4 CPU cores and 8GB of Memory that are available to docker. (Please
-    check the docker configuration if it provides sufficient resources to
-    docker.)
--   Network connection for downloading the container images for the network
-    components.
--   Command line terminal with a POSIX compliant shell.
--   `bash` installation.
--   [jq](https://jqlang.org).
+To shut down the network and remove containers:
 
-For deploying the Solidity contracts with hardhat:
+```sh
+./network devnet stop
+```
 
--   A local [Node.js](https://nodejs.org), [npm](https://www.npmjs.com) (or
-    yarn), and [npx](https://docs.npmjs.com/cli/v8/commands/npx) installation.
+## Integrating with other Hardhat projects
 
-For running the Blockscout instances:
+If you want to experiment with using the Chainweb EVM development network with other Hardhat projects, you must configure the Hardhat project to connect to the Chainweb EVM development network.
+You must also configure the Hardhat project to include account information—addresses and balances—for all available accounts.
+In the `kadena-evm-sandbox` directory, the `solidity` directory provides an example of a simple Hardhat project with a Hardhat configuration file, Solidity smart contract, and a test file.
 
--   At least 6 CPU cores and 12GB of RAM that are available to docker.
+The `solidity` project includes the following files to configure the Chainweb EVM development network:
 
-## Quick Start
+- The `solidity/devnet-accounts.json` file contains all of the account information generated from a test BIP-44 wallet using a seed entropy value of `0x0000 0000 0000 0000 0000 0000 0000 0000` (16 zero bytes).
+- The `solidity/hardhat.config.js` file reads the account information from the `solidity/devnet-accounts.json` file, sets the properties that describe the Chainweb EVM nodes, and configures the `kadena_devent` network as default Hardhat network for the `solidity` project. 
```

Comment:
This isn't actually configured right now because the line 52 `defaultNetwork: "kadena_devnet0",`has been commented out. That line should be uncommented and the text here should read `kadena_devnet0`. I don't know when or why that was commented out, but someone commented it out after I set up the config for the kadena devnet chains with kadena_devnet0 as the default. If this is uncommented, then I will need to make a small change in the test files, but I think it would be an improvement over not having the default set.